upgrade/rendering-changes/3-4-0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 3.4.0
+description: Handle the rendering changes in the 3.4.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in 3.4.0
+slug: changes-in-3-4-0
+position: 1
+---
+
+# Rendering Changes in 3.4.0
+
+## "k-state" CSS classes
+
+Previous to the 3.4.0 version we used `k-state-*` CSS classes to denote that the state of an HTML element happened. As part of the 3.4.0 release, we remove the `state` segment of that classes. 
+
+>caption A list of all changed "k-state" classes
+
+| Pre-3.4.0        | 3.4.0      |
```

Comment:
Didn't we change some of these for 3.0?

Reply:
I have seen an issue for 3.4.0 where I took the list of classes. Maybe I must investigate where exactly these classes were changed.